include/geos/geom/Coordinate.inl

```diff
@@ -157,6 +139,14 @@ operator!=(const Coordinate& a, const Coordinate& b)
     return ! a.equals2D(b);
 }
 
+INLINE size_t
+Coordinate::HashCode::operator()(const geos::geom::Coordinate &c) const {
+    size_t h = std::hash<double>{}(c.x);
+    h ^= std::hash<double>{}(c.y) << 1;
```

Comment:
Can you explain why you are shifting the hash here (and the c.z too)? Does that eliminate a bit of information?

Reply:
Just following the example at https://en.cppreference.com/w/cpp/utility/hash. There may be a better way to combine these, but I don't think our performance is terribly sensitive to the hash function.

Reply:
:ok_hand: Thank you, I was just curious.